definitions/features/pulp2.rb

```diff
@@ -34,4 +34,8 @@ def config_files
       '/etc/default/pulp_workers'
     ]
   end
+
+  def exclude_for_backup
+    ['var/lib/pulp/katello-export', 'var/lib/pulp/cache']
```

Comment:
The tar command is being run from `/var/lib/pulp` and hence exclude never worked. Request to change method like below,
~~~
def exclude_from_backup
  # Exclude /var/lib/pulp/katello-export and /var/lib/pulp/cache
  %w[katello-export cache]
end
~~~

You will also need to change `feature(:pulp2).exclude_for_backup` to `feature(:pulp2).exclude_from_backup`